include/color_spinor_field_order.h

```diff
@@ -858,6 +858,8 @@ namespace quda
       static constexpr int nSpin = nSpin_;
       static constexpr int nColor = nColor_;
 
+      using store_type = storeFloat;
```

Comment:
I believe this should be `store_t` to match the `real_t`, etc, convention we're slowly working towards---but if there's precedent for `store_type` elsewhere I won't fight too hard about this.

Reply:
https://github.com/lattice/quda/commit/d0da82373faf185f7b11fee05b7415cdbecf8424